src/components/MyRequest.vue

```diff
@@ -0,0 +1,39 @@
+<template>
+  <article class="request">
+    <header>
+      <h2>{{request.title}}</h2>
+    </header>
+    <p>
+      <strong>Kategorie:</strong> {{request.category.name}}
+    </p>
+    <p>
+      <strong>Gesuch:</strong> {{request.description}}
+    </p>
+  </article>
+</template>
+
+<style lang="sass" scoped>
+h2
+  color: $secondary
+  margin: 0
+  flex-shrink: 1
+
```

Comment:
Hmm.. maybe it's not good to change the `h2` all the time instead have a common `h2`